README.md

```diff
@@ -13,25 +13,22 @@ As such, the purpose of OpenIPSL are to be used as a research library (therefore
 ## Notice to Users:
 This project encompasses the development of a Modelica library for power system simulation.
 As such, there are no solver tools provided in this repository (no Power Flow Solvers, no Time-Domain solvers, etc.).
-You can find more information in our [**documentation**](http://openipsl.readthedocs.io/en/latest/get-started.html).
+You can find more information in our [**documentation**](http://openipsl.readthedocs.io/en/latest/user-guide/get-started.html).
```

Comment:
In the doc tree, I moved the getting started page in a sub-folder, which is why the link had changed. It is now updated